src/drivers/intel/cavs/hda-dma.c

```diff
@@ -860,11 +860,17 @@ static int hda_dma_remove(struct dma *dma)
 
 static int hda_dma_avail_data_size(struct hda_chan_data *chan)
 {
+	uint32_t status;
 	int32_t read_ptr;
 	int32_t write_ptr;
 	int size;
 
-	if (!(host_dma_reg_read(chan->dma, chan->index, DGCS) & DGCS_BNE))
+	status = host_dma_reg_read(chan->dma, chan->index, DGCS);
+
```

Comment:
No need to havev a new line here.

Reply:
We are doing double check, so in my opinion it's more readable with empty line between.